costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -99,7 +100,7 @@ void InflationLayer::onInitialize()
 
 void InflationLayer::reconfigureCB(costmap_2d::InflationPluginConfig &config, uint32_t level)
 {
-  setInflationParameters(config.inflation_radius, config.cost_scaling_factor);
+  setInflationParameters(config.inflation_radius, config.cost_scaling_factor, config.inflate_unknown);
 
   if (enabled_ != config.enabled) {
```

Comment:
Add to the above line to force reinflation if the inflate_unknown value changes.